documentation/docs/04-advanced-routing.md

```diff
@@ -112,7 +120,8 @@ SvelteKit needs to know which route is being requested. To do so, it sorts them
 src/routes/foo-abc/+page.svelte
 src/routes/foo-[c]/+page.svelte
 src/routes/[a]/+server.js
-src/routes/[b]/+page.svelte
+src/routes/[c]/+page.svelte
+src/routes/[[b]]/sub/+page.svelte
```

Comment:
just thinking out loud to myself... is this what we want? intuitively i feel like `/sub` should match `[[b]]/sub` before it matches `[c]`. hmm

Reply:
I think I'd agree with Rich at least for that one example. I'm not sure if there's an example motivating the version documented here